gradlew

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 #
-# Copyright 2021 Esri.
+# Copyright 2023 Esri.
```

Comment:
How did this file get an ESRI copyright in it? Isn't this fiel generated by Gradle?

Other `gradlew` files I've looked at have the following
```
#
# Copyright © 2015-2021 the original authors.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#      https://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
```
and in the file it also has
```
Gradle start up script for POSIX generated by Gradle.
```

Reply:
We were requested to make this change by gitadmin group when we first published the repository.  However I'm very happy to revert it to the default as its a pain to update every time we update the gradle wrappers.